gizmo.py

```diff
@@ -0,0 +1,93 @@
+import numpy
+import pandas
+
+def hello(name, country="Finland"):
+    
+    print("Hello %s, how are things in %s?" % (name, country))
+    
+def spell(word):
+    
+    for i in range(0, len(word)):
+        print(word[i], end = '')
+        if i != len(word)-1:
+            print(".", end = '')
+
+def relative_path(subject_identifier):
+    names = []
+    for i in subject_identifier:
```

Comment:
`i` is usually reserved for integer counters. I would personally do something like:

```python
for id in subject_identifiers:
```